src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -36,30 +37,71 @@ import scala.annotation.tailrec
  * For more details @see https://en.wikipedia.org/wiki/Kolakoski_sequence
  */
 
-object Homework :
+object Homework:
 
-  object `Boolean Operators` :
+  object `Boolean Operators`:
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = if b then false else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean = if left then right else false

```

Comment:
Since we reproduce "Java" eager operation, then we should prove that if evaluation of `right` if not necessary then `right` should not be evaluated.
Hint: "call-by-name".

Reply:
Please apply the comment above.